pkg/charts/checkRCCharts.go

```diff
@@ -10,14 +9,7 @@ import (
 )
 
 // CheckRCCharts checks for any charts that have RC versions
-func CheckRCCharts() map[string][]string {
-
-	// Get the current working directory
-	repoRoot, err := os.Getwd()
-	if err != nil {
-		logrus.Fatalf("Unable to get current working directory: %s", err)
-	}
-
+func CheckRCCharts(repoRoot string) map[string][]string {
```

Comment:
One of the main reasons, I've removed os.Getwd() definitions in package functions is to get rid of scenarios where all functions are not sharing the same abstraction for the repoRoot -- in the case where we make changes like this one.

In this PR I'm preferring using getRepoRoot() function defined in main.go and passing it through to sub functions as that seemed to be mostly the standard when Arvind was still working on the project

Reply:
We could also have a shared variable in pkg/charts, along the lines of:

```go
func init() {

  repoRoot, err := os.Getwd()
  if err != nil {
    logrus.Fatal(err)
  } 
  DefaultRepoRoot= repoRoot
}

var (
  // default value
  DefaultRepoRoot = ""
  // flag writes to this one
  RepoRoot = ""
)
```

which could get passed in / written to as a cli.Flag, and consumers would use as `charts.RepoRoot`, but I opted for simplicity here